src/ovirtapi/transform.js

```diff
@@ -68,6 +68,28 @@ function cleanUndefined (obj: Object): Object {
   return obj
 }
 
+const colors = [
+  '#ec7a08',
+  '#f0ab00',
+  '#92d400',
+  '#3f9c35',
+  '#007a87',
+  '#00b9e4',
+  '#703fec',
+  '#486b00',
+  '#003d44',
+  '#005c73',
+  '#40199a',
+]
+
+function getPoolColor (id: string): string {
+  let poolColor = 0
+  for (let i = 0; i < id.length; i++) {
+    poolColor += id.charCodeAt(i)
+  }
+  return colors[poolColor % colors.length]
```

Comment:
The main reason of this solution is that user must associate color with particular pool, so it is better to assign somehow color to pool, so as even user reload page pool will has same color. The better way for this is to calculate color based on id, because id, unlike to name, never change.

Reply:
Good point.